contrib/opencensus-ext-django/tests/test_django_middleware.py

```diff
@@ -25,7 +25,7 @@
 from opencensus.trace.exporters import print_exporter
 from opencensus.trace.exporters import zipkin_exporter
 from opencensus.trace.exporters import jaeger_exporter
-from opencensus.trace.exporters.ocagent import trace_exporter
+from opencensus.ext.ocagent import trace_exporter
```

Comment:
Looks like we need to decouple exporters from framework integrations.

https://github.com/census-instrumentation/opencensus-python/blob/80455edc4a461440caaa7cfd0bff873ea33a8244/contrib/opencensus-ext-django/opencensus/ext/django/middleware.py#L43